zebra-chain/src/parameters/network_upgrade.rs

```diff
@@ -274,6 +280,16 @@ impl NetworkUpgrade {
     ) -> Duration {
         NetworkUpgrade::current(network, height).averaging_window_timespan()
     }
+
+    /// Always returns true if `network` is the Mainnet.
```

Comment:
Let's follow the pattern used by the other functions:
1. Short summary of what the function does
2. Blank line
3. Detailed explanation of what the function does


```suggestion
    /// Returns true if the maximum block time rule is active for `network` and `height`.
    ///
    /// Always returns true if `network` is the Mainnet.
```